test/spec/services/apiServiceSpec.js

```diff
@@ -234,6 +235,96 @@ describe("APIService", function(){
       });
     }));
   });
-  */
+
+
+  describe('#availableKinds', function() {
+    var bothSample = ['Binding','ConfigMap','DeploymentConfig','Event','LimitRange','Pod','ReplicaSet','Role','Service', 'Template'];
+    var onlyClusterSample = ['ClusterResourceQuota','Namespace','OAuthAccessToken','PersistentVolume','ProjectRequest','User'];
+
+    // at minimum, prove with tests that it is deduplicating by ignoring openshift!
+    it('should return list of kinds that are scoped to a namespace by default', function() {
+      var namespacedKinds = _.map(APIService.availableKinds(), 'kind');
+      // from the fixtures included, we should have 51 namespaced kinds
```

Comment:
can you update the comments, they are still based on your old version of the test